src/server/rdb_extensions.h

```diff
@@ -8,8 +8,8 @@ extern "C" {
 #include "redis/rdb.h"
 }
 
-//  Custom types: Range 20-25 is used by DF RDB types.
-const uint8_t RDB_TYPE_JSON = 20;
+//  Custom types: Range 30-35 is used by DF RDB types.
```

Comment:
Would this break existing Dragonfly snapshots?

Reply:
Big yes :( But if we don't, we won't be able to support the current and newest RDB formats

Reply:
but we also parse RDB_VERSION. We could at least support loading of the older RDB format with `RDB_TYPE_JSON=20`

Reply:
Yes we can, I added it -- really good suggestion.